web/components/view/ShowHideTextView/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import arrow from '../../../assets/arrow.svg';
+
+class ShowHideTextView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {isToggleOn: true};
```

Comment:
I think `isToggleOn` is pretty vague? I have to read the code to figure out whether `isToggleOn = true` means the dropdown is shown or not. Could we use something different like `showDropDown`?